server/main.py

```diff
@@ -97,4 +303,4 @@ async def make_move(game_id: int, move: schemas.Move):
 if __name__ == "__main__":
     import uvicorn
 
-    uvicorn.run("main:app", host="0.0.0.0", port=8000, reload=True)
+    uvicorn.run("main:app", host="127.0.0.1", port=8000, reload=True)
```

Comment:
`0.0.0.0` should work even if your server running not on the same machine as client. `127.0.0.1` won't work like that.